daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_nvidia_latest.publish.json

```diff
@@ -35,7 +35,7 @@
     {
       "Prefix": "rocky-linux-8-optimized-gcp-nvidia-latest",
       "Family": "rocky-linux-8-optimized-gcp-nvidia-latest",
-      "Description": "Rocky Linux, Rocky Linux, 8 optimized for GCP, x86_64 optimized for GCP with latest Nvidia driver built on {{$time}}",
+      "Description": "Rocky Linux, Rocky Linux, 8 optimized for GCP, x86_64 optimized for GCP with Nvidia driver 570 built on {{$time}}",
```

Comment:
the 570 Nvidia driver

seems like the better format for consistency and it sounds more accurate when I say the statement aloud.

If you agree, it needs to be changed throughout the files. If you disagree I'm fully open to other prospectives.

Reply:
No real opposition here so I'll change it